### DISCUSSION TITLE
cannot import name 'vtkExtractEdges' from 'vtkmodules.vtkFiltersExtraction'

### DISCUSSION THREAD

**Discussion by Ali-Iftikhar** [pyvista/pyvista#2858] [Q&A] [answered]
Dear All
I installed pyvista using on my laptop with Ubuntu (version 22) by using the command "pip install pyvista". But when I import pyvista, it gives the following error.

ImportError: cannot import name 'vtkExtractEdges' from 'vtkmodules.vtkFiltersExtraction' (/home/ifti/.local/lib/python3.10/site-packages/vtkmodules/vtkFiltersExtraction.cpython-310-x86_64-linux-gnu.so)

Need your help. Thanks

**Comment by MatthewFlamm** [comment, marked as answer]
See #2859.  One workaround for the time being is to install a custom wheel from https://github.com/pyvista/pyvista-wheels.

Another workaround is to install python 3.9, if you can.
